docs/features/policy.md

```diff
@@ -0,0 +1,37 @@
+# OPA Policies
+
+The [OPA Report](/reports/opa) allows you to define custom policies for checking Kubernetes resources.
+This is useful for enforcing policies that are specific to your organization - e.g. particular labeling
+schemes or required annotations.
+
+This page discusses the Policy UI - to learn more about how to write OPA
+policies you can check out the [OPA Documentation](/reports/opa)
+
+## Create a Policy
+To get started, go to the `Policy` tab, and click the `Create Custom Policy` button.
+
+<div>
+  <img :src="$withBase('/img/policy.png')" alt="create policy">
+</div>
+
+First, be sure to give your policy a good name and description, so you can refer back to it later.
```

Comment:
The instance name field I missed at first, so it could be especially good to call that out in these docs.